### ISSUE TITLE
Add data from CDC funded nursing home study https://github.com/abtassociates/CDC_NHPHRN

### ISSUE THREAD

**Issue by YWAN446** [shedding-hub/shedding-hub#70]
We should contact the CDC team to check whether it is OK to include this on Shedding Hub directly without a data use agreement.

FYI, the CDC funded Nursing Home Public Health Response Network is making its COVID-19 kinetics data publicly available, so CDC has already analyzed these, and I think one manuscript is in submission, but in case there is use for re-use of these data by your group, just wanted to be sure you knew about it. CDC is trying to priorizie public use of data as you know…maybe even use in your classroom activity….
De-identified INHERENT 1.0 Public Use Files are now available on Abt’s Github site https://github.com/abtassociates/CDC_NHPHRN.
Basically - we (then network) swabbed SARS-CoV-2 exposed workers and residents of 25 or so nursing homes as fast as we identified them, then if + swabbed them as close to daily as we could for 2 weeks, some for 3 months (weekly) to look for shedding, positivity, rebound, etc, and CRFs of their history and vaccine status.
